opencadd/structure/superposition/engines/theseus.py

```diff
@@ -117,7 +135,6 @@ def _calculate(self, structures, *args, **kwargs) -> dict:
                     ``d_p``: d_p
                     ``median_structure``: median_structure
                     ``n_total``: number total
-                    ``n_atoms``: number of atoms
```

Comment:
Removed because you only work with Ca atoms?

Reply:
`n_total` refers to what?
Could you please extend the description in the docstring, currently only contains

```
``n_total``: number total
```

Reply:
'n_atoms' are not removed. It is now in 'scores' as 'coverage'.
'n_total' is the total number of aligned atoms. (2 times 'n_atoms')

Description will be updated.